docs/FAQ.md

```diff
@@ -0,0 +1,123 @@
+
+
+## General
+
+### How can I find you on uniswap?
+Yield Protocol has no governance token. You will not find us on Uniswap, and you should be wary about any token claiming to be from Yield Protocol. 
+
+You may find the fyDai or fyDaiLP tokens created by the protocol on Uniswap or other exchanges since they are ERC-20 assets. We recommend using those assets with the [Yield App](https://app.yield.is) since the app is tailored to the particularities of trading those assets. 
+
+### When will you release a governance token?
+There are no current plans to release a governance token.
+
+### Does the Yield Protocol have developer fees?  
+Yield Protocol v1 does not have developer fees. Users of the Yield App will pay fees for borrowing and lending via their use of the YieldSpace liquidity pools. All fees in the liquidity pools go to liquidity providers. 
+
+### How is APR calculated?
+
+The APR for various series in the Yield App are set by automated liquidity pools. The interest rate (and price) that an automated liquidity pool trades at is determined by the current reserves of fyDai and Dai held in the pool. The mid-market APR (the average of the borrowing and lending APRs) can be calculated with the following formula: 
+
+$$
+(\frac{fyDaiReserves}{DaiReserves})^\frac{1}{4} - 1
+$$
+
+### Fee in the AMM is 0.3% right? Like Uniswap's?
+No, it’s calculated differently. It’s approximately 5% of the interest paid. You can read about fees in the [YieldSpace paper](https://yield.is/YieldSpace.pdf) or in these docs.
+
+### What is the fee structure? Formula/parameters?
+In other protocols, fees are typically charged in proportion to the amount being sold.  In the Yield App liquidity pools, prices are computed in “yield space” rather than “price space”—meaning that the fees impose a proportional spread on interest rates, rather than on prices. The exact fee calculation is detailed in our [YieldSpace paper](https://yield.is/YieldSpace.pdf) in Section 5. As a rough approximation, the fee is about 5% of the current interest rate. For example, when the marginal interest rate of an automated liqiduity pool is 10%, a borrower might see a marginal interest rate of ≈10.55%, while a lender might see a marginal interest rate of ≈9.47%.
+
+### When telegram?
+We have a thriving community in [Discord](https://discord.com/channels/752978124614008945), join us there!
+
+### What assets can be borrowed or lent? What collateral can be used. 
+The Yield Protocol v1 only permits borrowing and lending of Dai, and only accepts ETH and Chai collateral. Chai collateral is not currently supported in the Yield App. 
+
+### Is this built with UMA?
+Yield is not built with UMA. It is built on top of Maker.
+
+### Are there any major differences between fyDai and UMA's uUSD?
+The main differences are the collateral assets, the settlement assets, and the pool mechanics.
```

Comment:
Clayton answered this in a good way: 


When UMA's uUSD expires, the outstanding uUSD becomes "de-pegged", and redeemable for the underlying collateral: ETH.

When Yield Protocol's fyDai expires it remains as Dai, but the fixed rate becomes a variable rate. This could be seen as "opening a CDP on the borrower's behalf."